### PR TITLE
fixed the icon lmao

### PR DESCRIPTION
**Pull request by BlaNKtext**
sorry for formatting btw

### CODE REVIEW

**Comment by Yasamato** [comment]
Dude, I don't want to be rude, but please refrain from autoformatting the whole index with your different code-style...

Besides why do you want to add spaces in strings like `rel="stylesheet "` or here `width="32 " height="32 "`?

I've added a favicon-pregenerated preset in https://github.com/ranimepiracy/index/commit/53f8d8699c230fceafc9b49a67b033d39d323c6b and as I see no real changes or benefits I'll close this PR. But feel free to contribute any other worthwhile changes 👍

File: index.html [hunk 1]
```diff
@@ -2,262 +2,236 @@
...
 
 <head>
-    <meta charset="utf-8"/>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
-    <link rel="shortcut icon" type="image/png" href=”/img/logo.png”/>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="img/logo.png">
```
Comment (Yasamato) [inline index.html:8]: see https://www.w3schools.com/tags/att_link_sizes.asp, but yeah why was there shortcut?

**Comment by BlaNKtext** [comment]
> Dude, I don't want to be rude, but please refrain from autoformatting the whole index with your different code-style...
> 
> Besides why do you want to add spaces in strings like `rel="stylesheet "` or here `width="32 " height="32 "`?


Yeah sorry about that, I realize that makes it hard to use pull requests, If I continue to add PRs I'll disable my autoformatting

File: index.html (hunk 1 above)
Comment (BlaNKtext) [reply index.html:8]: I'm not sure, I def gotta change some VS code settings
Thank you for bringing my screwed up fomatting to light

**Comment by Yasamato** [comment]
You should definetly use autoformatting, but you can adjust the project-setting in vs.

Doing formatting by hand is really not the way to go...

**Pull request closed without merging**